docs/menu.yml

```diff
@@ -203,7 +203,11 @@ catalog:
           - name: "Kubernetes"
             catalog:
               - name: "Observe Kubernetes"
-                path: "/en/setup/backend/backend-k8s-monitoring"
```

Comment:
`/en/setup/backend/backend-k8s-monitoring` is referenced by blogs and SkyWalking UI, it should not be changed.

We should build a summary introduction doc for this path. And cAdvisor and eBPF should be two documents more about details.

Reply:
I have created a new summary document and linked these two documents together.